src/Stepper/Stepper.less

```diff
@@ -0,0 +1,24 @@
+@import "../less/fonts.less";
+@import (reference) "../less/index";
+
+.Stepper {
+  .flexbox();
+}
+
+.Stepper--sidebar {
```

Comment:
related to last review - move sidebar stuff into a higher-order component if needed.

ideally, we'd just have some generic sticky sidebar container component that's very simple and just renders the content it's given and handles positioning with a `maxWidth` prop